metrics/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.22 AS builder
```

Comment:
As a follow-on, it might be nice to set up Dependabot in this repo to automate PRs like these